docs/sources/mimir/references/configuration-parameters/index.md

```diff
@@ -1033,6 +1033,10 @@ instance_limits:
 # (experimental) Enable logging of utilization based limiter CPU samples.
 # CLI flag: -ingester.log-utilization-based-limiter-cpu-samples
 [log_utilization_based_limiter_cpu_samples: <boolean> | default = false]
+
+# (experimental) Use experimental method of limiting push requests
+# CLI flag: -ingester.limit-inflight-requests-using-grpc-handlers
+[limit_inflight_requests_using_grpc_tap_handle: <boolean> | default = false]
```

Comment:
YAML parameter and CLI flag don't match. Can you fix it, please?

Reply:
Sure

Reply:
Will be fixed in https://github.com/grafana/mimir/pull/6073 (draft for now)